framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.7 under development
 -----------------------
 
+- Enh #5469: Add mimetype validation by mask in FileValidator
```

Comment:
i.e. it should be moved to all other lines with `Enh` type and should be right after issue with the number that's less than its own.